includes/class-wc-payments-account.php

```diff
@@ -1588,12 +1588,12 @@ private function maybe_redirect_to_treatment_onboarding_page() {
 	}
 
 	/**
-	 * Redirects to the onboarding prototype page if the feature flag is enabled.
+	 * Redirects to the onboarding flow page if the Progressive Onboarding feature flag is enabled.
 	 * Also checks if the server is connect and try to connect it otherwise.
 	 *
 	 * @return void
 	 */
-	private function redirect_to_prototype_onboarding_page() {
+	private function redirect_to_onboarding_flow_page() {
```

Comment:
There's also another [place](https://github.com/Automattic/woocommerce-payments/blob/c5a2776b934ab597c9e6a3c8b455dc723c448475/includes/class-wc-payments-account.php#L1601-L1601) where the redirect happens, when the user initiates onboarding via the `Payments` page on WC Admin. 🙂 That link was still pointing to the old prototype URL, but it worked well after I tested updating it.

Reply:
Oh yes, how I missed it! Changed here https://github.com/Automattic/woocommerce-payments/pull/6332/commits/426d918018f6b4a2c71b52aa65feeca2f2e34030